### PR TITLE
Added scipy.stats.gennorm.

### PR DESCRIPTION
**Pull request by carlosgmartin**

### CODE REVIEW

File: jax/scipy/stats/gennorm.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Google LLC
```
Comment (jakevdp) [inline jax/scipy/stats/gennorm.py:1]: 2022

File: jax/_src/scipy/stats/gennorm.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Copyright 2018 Google LLC
```
Comment (jakevdp) [inline jax/_src/scipy/stats/gennorm.py:1]: 2022

**Review by jakevdp — CHANGES_REQUESTED** [review]
Looks great! Just minor comments

**Review by jakevdp — APPROVED** [review]

**Comment by jakevdp** [comment]
Merging failed because the new tests are failing on TPU due to floating point precision. We should be able to fix it by bumping the `rtol` in `CheckAgainstNumpy` from 0.0001 to 0.001

**Comment by jakevdp** [comment]
Hi - just taking another look at this. Can you squash the two commits and rebase on the current main branch? Thanks!

**Review by jakevdp — APPROVED** [review]

**Comment by jakevdp** [comment]
Looks like there are some conflicts... can you rebase against the updated main branch?

**Comment by carlosgmartin** [comment]
@jakevdp Done.

**Comment by jakevdp** [comment]
Thanks!

**Pull request merged**